include/seqan3/core/algorithm/concept.hpp

```diff
@@ -57,19 +57,15 @@ class config_element_base;
 /*!\var id
  * \brief Algorithm specific static id used for internal validation checks.
  */
-/*!\var value
- * \brief Member storing the configuration value.
- */
 //!\}
 //!\cond
 template <typename config_t>
 SEQAN3_CONCEPT config_element = std::semiregular<std::remove_reference_t<config_t>> &&
 requires (config_t c)
 {
     { std::remove_reference_t<config_t>::id };
-    { c.value };
     // Must inherit from the pipeable_config_element class.
-    requires std::is_base_of_v<pipeable_config_element<config_t, decltype(c.value)>, config_t>;
+    // requires std::is_base_of_v<pipeable_config_element<config_t, decltype(c.value)>, config_t>;
```

Comment:
Mhm, I'm not entirely sure how to handle this; we probably need to expose the value_t of the configuration?

Reply:
Yes, I was thinking the same here, but that almost feels like the old design... Is `value_t` actually used in the new design or is its purpose only the concept check? My hope was that after refactoring we could just include this line again without the decltype part.

Reply:
No, just remove it.